### DISCUSSION TITLE
Sub group/list for children

### DISCUSSION THREAD

**Discussion by CFox243** [cmintey/wishlist#194] [Ideas]
For parents who are adding items for kids, it would be nice to be able to separate the parents' list from the children's wishlist without needing an email account/or needing to sign in to another account to make their list/edit the list.

Ability to make and name a (x) list with name. Then add items to that list like normal.
Maybe add permissions to the custom list to be able to open the list to the group for editing or just the creator.

Also thank you for your work on this Project. it solves a lot of issues with planning who gets what for birthdays/holidays.

**Comment by CFox243** [comment]
i see that this was posted as a request under issues ([Link](https://github.com/cmintey/wishlist/issues/148#issue-2632705527))
so, a slight change adds an account without the need for an email address?

**Reply by cmintey** [reply]
Well you still need an email to create an account to do what I described in that issue. However, I am currently working on adding support for multiple lists under a single account, which would work exactly as you described. Hopefully it'll be done soon, but with the holidays coming up, might not be until after then. Thanks for the suggestion!

**Reply by CFox243** [reply]
Thank you for your time. and i understand enjoy your holidays.
